# Hungarian translation of update-manager
# This file is distributed under the same license as the update-manager package.
# Copyright (C) 2005, Free Software Foundation, Inc.
# Gabor Kelemen <kelemeng@gnome.hu>, 2005.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager.HEAD\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2009-04-07 08:37+0000\n"
"PO-Revision-Date: 2009-04-07 10:29+0000\n"
"Last-Translator: Máté Őry <orymate@gmail.com>\n"
"Language-Team: Hungarian <gnome@gnome.hu>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2009-04-08 12:12+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:233 ../UpdateManager/Core/utils.py:233
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:236 ../UpdateManager/Core/utils.py:236
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:239 ../UpdateManager/Core/utils.py:239
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:242 ../UpdateManager/Core/utils.py:242
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:207 ../DistUpgrade/distro.py:422
#, python-format
msgid "Server for %s"
msgstr "Kiszolgáló a következőhöz: %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:225 ../DistUpgrade/distro.py:231
#: ../DistUpgrade/distro.py:247
msgid "Main server"
msgstr "Fő kiszolgáló"

#: ../DistUpgrade/distro.py:251
msgid "Custom servers"
msgstr "Egyéni kiszolgálók"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "Nem határozható meg a sources.list bejegyzés"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Egyetlen csomagfájl sem található! Lehet, hogy nem Ubuntu lemezt vagy pedig "
"más architektúrához tartozót helyezett a meghajtóba."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "A CD hozzáadása meghiúsult"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Hiba történt a CD hozzáadása során, a frissítés félbeszakad. Jelentse ezt "
"hibaként, ha ez egy érvényes Ubuntu CD.\n"
"\n"
"A hibaüzenet:\n"
"„%s”"

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Hibás állapotban levő csomag eltávolítása"
msgstr[1] "Hibás állapotban levő csomagok eltávolítása"

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"A(z) „%s” nevű csomag hibás! Újratelepítése szükséges, azonban nem található "
"hozzá megfelelő archívum. El kívánja távolítani ezt a csomagt a folytatáshoz?"
msgstr[1] ""
"A(z) „%s” nevű csomagok hibásak! Újratelepítésük szükséges, azonban nem "
"található hozzájuk megfelelő archívum. El kívánja távolítani ezeket a "
"csomagokat a folytatáshoz?"

#: ../DistUpgrade/DistUpgradeCache.py:330
msgid "Broken packages"
msgstr "Törött csomagok"

#: ../DistUpgrade/DistUpgradeCache.py:331
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"A rendszere törött csomagokat tartalmaz, amelyek ezzel az alkalmazással nem "
"javíthatóak. Kérem, először javítsa ki őket a synaptic vagy az apt-get "
"segítségével."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:564
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Megoldhatatlan probléma adódott a frissítés előkészítése közben:\n"
"%s\n"
"\n"
" Ezt az alábbiak okozhatták:\n"
" * frissítés egy kiadás előtti Ubuntu verzióra\n"
" * Ön jelenleg egy kiadás előtti Ubuntu verziót használ\n"
" * az Ubuntu által nem támogatott csomagok\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:574
msgid "This is most likely a transient problem, please try again later."
msgstr "Ez valószínűleg átmeneti probléma, próbálkozzon később."

#: ../DistUpgrade/DistUpgradeCache.py:577
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Ha ezek közül egyik sem teljesül, jelentse ezt az „update-manager” csomaggal "
"kapcsolatos hibát és mellékelje a /var/log/dist-upgrade/ könyvtárban "
"található fájlokat a hibajelentéshez."

#: ../DistUpgrade/DistUpgradeCache.py:583
#: ../UpdateManager/UpdateManager.py:727
msgid "Could not calculate the upgrade"
msgstr "A frissítés előkészítése sikertelen"

#: ../DistUpgrade/DistUpgradeCache.py:633
msgid "Error authenticating some packages"
msgstr "Hiba történt néhány csomag hitelesítése közben"

#: ../DistUpgrade/DistUpgradeCache.py:634
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Néhány csomagot nem sikerült hitelesíteni. Lehetséges, hogy ezt átmeneti "
"hálózati probléma okozza, ezért érdemes később újra megpróbálni. Az alábbi "
"felsorolás a hitelesíthetetlen csomagokat tartalmazza."

#: ../DistUpgrade/DistUpgradeCache.py:654
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"A(z) '%s' nevű csomagot eltávolításra jelölte ki, de ez rajta van az "
"eltávolítási feketelistán."

#: ../DistUpgrade/DistUpgradeCache.py:658
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "A létfontosságú '%s' csomagot eltávolításra jelölte ki."

#: ../DistUpgrade/DistUpgradeCache.py:745
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "A(z) '%s' csomag nem jelölhető ki frissítésre."

#: ../DistUpgrade/DistUpgradeCache.py:759
#, python-format
msgid "Can't install '%s'"
msgstr "„%s” nem telepíthető"

#: ../DistUpgrade/DistUpgradeCache.py:760
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr "Egy szükséges csomag nem telepíthető. Kérem, jelentse ezt hibaként. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:768
msgid "Can't guess meta-package"
msgstr "A meta-csomag megállapítása sikertelen"

#: ../DistUpgrade/DistUpgradeCache.py:769
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"A rendszere nem tartalmazza az ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop vagy edubuntu-desktop csomagok egyikét sem, és nem lehetett "
"megállapítani, hogy az Ubuntu mely változatát használja.\n"
" A folytatás előtt telepítse a fenti csomagok egyikét a synaptic vagy az apt-"
"get használatával."

#: ../DistUpgrade/DistUpgradeController.py:85
msgid "Reading cache"
msgstr "Gyorsítótár beolvasása"

#: ../DistUpgrade/DistUpgradeController.py:178
msgid "Unable to get exclusive lock"
msgstr "A kizárolagos zárolás nem lehetséges"

#: ../DistUpgrade/DistUpgradeController.py:179
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Ez általában azt jelenti, hogy már fut egy másik csomagkezelő alkalmazás "
"(például apt-get vagy aptitude). Kérem először zárja be azt az alkalmazást."

#: ../DistUpgrade/DistUpgradeController.py:232
msgid "Upgrading over remote connection not supported"
msgstr "A frissítés nem támogatott távoli kapcsolaton keresztül"

#: ../DistUpgrade/DistUpgradeController.py:233
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. The upgrade will abort now. Please try without "
"ssh."
msgstr ""
"A frissítést távoli SSH kapcsolaton keresztül futtatja egy ezt nem támogató "
"előtétprogrammal. A frissítés most megszakad. Próbálja újra SSH nélkül."

#: ../DistUpgrade/DistUpgradeController.py:244
msgid "Continue running under SSH?"
msgstr "Fusson tovább SSH alatt?"

#: ../DistUpgrade/DistUpgradeController.py:245
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, a additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Úgy tűnik, hogy a frissítés SSH alatt fut.  Nem javasolt frissítést végezni "
"SSH alatt, mert ha hiba lépne fel, nehezebb azt helyrehozni.\n"
"\n"
"Ha folytatja, egy új SSH kiszolgáló indul a következő porton: %s.\n"
"Kívánja folytatni?"

#: ../DistUpgrade/DistUpgradeController.py:260
msgid "Starting additional sshd"
msgstr "Új sshd indítása"

#: ../DistUpgrade/DistUpgradeController.py:261
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Ahhoz, hogy  hiba esetén a helyreállítás könnyebb legyen, egy további sshd "
"indul a(z) %s porton. Ha bármiféle probléma felmerülne az aktuális ssh "
"futása során, Ön továbbra is kapcsolódni tud majd a másik segítségével.\n"

#: ../DistUpgrade/DistUpgradeController.py:331
#: ../DistUpgrade/DistUpgradeController.py:376
msgid "Can not upgrade"
msgstr "Nem lehet frissíteni"

#: ../DistUpgrade/DistUpgradeController.py:332
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""
"Ez az eszköz nem támogatja a frissítést „%s” rendszerről „%s” rendszerre."

#: ../DistUpgrade/DistUpgradeController.py:341
msgid "Sandbox setup failed"
msgstr "A tesztkörnyezet beállítása meghiúsult"

#: ../DistUpgrade/DistUpgradeController.py:342
msgid "It was not possible to create the sandbox environment."
msgstr "Nem sikerült létrehozni a tesztkörnyezetet."

#: ../DistUpgrade/DistUpgradeController.py:348
msgid "Sandbox mode"
msgstr "Teszt mód"

#: ../DistUpgrade/DistUpgradeController.py:349
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"A frissítés teszt módban fut. Minden módosítás kiírásra kerül a(z) %s fájlba "
"és a következő újraindításkor elvész.\n"
"\n"
"Mostantól a következő újraindításig a rendszerkönyvtárakba írt változtatások "
"NEM véglegesek."

#: ../DistUpgrade/DistUpgradeController.py:377
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"A Python telepítése sérült. Javítsa ki a „/usr/bin/python” szimbolikus "
"linket."

#: ../DistUpgrade/DistUpgradeController.py:403
msgid "Package 'debsig-verify' is installed"
msgstr "A „debsig-verify” csomag telepítve van"

#: ../DistUpgrade/DistUpgradeController.py:404
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"A frissítés nem folytatódhat, ha ez a csomag telepítve van.\n"
"Távolítsa el a Synaptic vagy az „apt-get remove debsig-verify” parancs "
"kiadásával és futtassa újra a frissítést."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid "Include latest updates from the Internet?"
msgstr "Használja az internetről elérhető legújabb frissítéseket?"

#: ../DistUpgrade/DistUpgradeController.py:415
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"A frissítőrendszer használhatja az internetet a legújabb frissítések "
"letöltéséhez és telepítheti azokat a frissítés közben. Ha Ön rendelkezik "
"internetkapcsolattal, ez a beállítás erősen ajánlott.\n"
"\n"
"A frissítés hosszabb ideig is eltarthat, a folyamat végeztével az Ön "
"rendszere teljesen naprakész lesz. Választhatja e funkció mellőzését is, "
"azonban ajánlott a legújabb frissítések telepítését elvégezni a "
"rendszerfrissítés után  nem sokkal.\n"
"Ha a „Nem” gombot választja, a telepítő nem fogja használni "
"internetkapcsolatát."

#: ../DistUpgrade/DistUpgradeController.py:626
#, python-format
msgid "disabled on upgrade to %s"
msgstr "A(z) '%s' nevű csomag nem frissíthető."

#: ../DistUpgrade/DistUpgradeController.py:652
msgid "No valid mirror found"
msgstr "Nem található érvényes tükör"

#: ../DistUpgrade/DistUpgradeController.py:653
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found.This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'no' the update will cancel."
msgstr ""
"A tárolóinformációk elemzése során a program nem talált tükörbejegyzést a "
"frissítéshez. Ez akkor fordulhat elő, ha belső tükröt használ, vagy a "
"tükörinformációk elavultak.\n"
"\n"
"Mindenképpen újra kívánja írni a sources.list fájlt? Ha az „Igen” gombot "
"választja, akkor az összes „%s” bejegyzés „%s” bejegyzéssé lesz frissítve. \n"
"A „Nem” kiválasztása megszakítja a frissítést."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:673
msgid "Generate default sources?"
msgstr "Alapértelmezett források ismételt előállítása?"

#: ../DistUpgrade/DistUpgradeController.py:674
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the update "
"will cancel."
msgstr ""
"A „sources.list” átnézése után sem található érvényes bejegyzés a "
"következőről: %s.\n"
"\n"
"Hozzáadja az alap bejegyzéseket a következőhöz: „%s”? A „Nem” kiválasztása "
"megszakítja a frissítést."

#: ../DistUpgrade/DistUpgradeController.py:709
msgid "Repository information invalid"
msgstr "Érvénytelen tárolóinformáció"

#: ../DistUpgrade/DistUpgradeController.py:710
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"A tárolóinformáció frissítése hibás fájlt eredményezett. Kérem, jelentse ezt "
"hibaként."

#: ../DistUpgrade/DistUpgradeController.py:716
msgid "Third party sources disabled"
msgstr "A külső források letiltva"

#: ../DistUpgrade/DistUpgradeController.py:717
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"A sources.list néhány harmadik féltől származó forrása le lett tiltva. A "
"frissítés után újraengedélyezheti őket a Szoftverforrások eszközzel, vagy a "
"csomagkezelővel."

#: ../DistUpgrade/DistUpgradeController.py:758
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Nem konzisztens állapotú csomag"
msgstr[1] "Nem konzisztens állapotú csomagok"

#: ../DistUpgrade/DistUpgradeController.py:761
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"A(z) „%s” nevű csomag hibás! Újratelepítése szükséges, azonban nem található "
"hozzá megfelelő archívum. Telepítse újra a csomagot saját kezűleg, vagy "
"távolítsa el a rendszerről."
msgstr[1] ""
"A(z) „%s” nevű csomagok hibásak! Újratelepítésük szükséges, azonban nem "
"található hozzájuk megfelelő archívum. Telepítse újra a csomagot saját "
"kezűleg, vagy távolítsa el a rendszerről."

#: ../DistUpgrade/DistUpgradeController.py:809
msgid "Error during update"
msgstr "Hiba történt a frissítés közben"

#: ../DistUpgrade/DistUpgradeController.py:810
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Hiba lépett fel a frissítés közben. Ez többnyire hálózati problémára utal. "
"Kérem, ellenőrizze a hálózati kapcsolatot és próbálja újra."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:819
#: ../UpdateManager/UpdateManager.py:563
msgid "Not enough free disk space"
msgstr "Nincs elég szabad hely a merevlemezen"

#: ../DistUpgrade/DistUpgradeController.py:820
#, python-format
msgid ""
"The upgrade is now aborted. The upgrade needs a total of %s free space on "
"disk '%s'. Please free at least an additional %s of disk space on '%s'. "
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"A frissítés most megszakadt. A frissítéshez %s szabad hely szükséges a(z) "
"„%s” lemezen. Szabadítson fel további %s lemezterületet a(z) „%s” eszközön. "
"Ürítse a Kukát és törölje a korábbi telepítések átmeneti fájljait a „sudo "
"apt-get clean” parancs kiadásával."

#: ../DistUpgrade/DistUpgradeController.py:860
msgid "Support for some applications ended"
msgstr "Egyes alkalmazások támogatása megszűnt"

#: ../DistUpgrade/DistUpgradeController.py:861
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"If you have not enabled community maintained software (universe), these "
"packages will be suggested for removal at the end of the upgrade."
msgstr ""
"A Canonical Ltd. már nem biztosít támogatást a következő "
"szoftvercsomagokhoz. A közösségtől továbbra is kaphat támogatást. \n"
"\n"
"Ha nincsenek engedélyezve a közösség által karbantartott szoftverek "
"(universe tároló), akkor ezek a csomagok a telepítés végén eltávolításra "
"lesznek javasolva."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:870
#: ../DistUpgrade/DistUpgradeController.py:1529
msgid "Calculating the changes"
msgstr "Módosítások előkészítése…"

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:885
msgid "Do you want to start the upgrade?"
msgstr "Biztosan el szeretné kezdeni a frissítést?"

#: ../DistUpgrade/DistUpgradeController.py:937
#: ../DistUpgrade/DistUpgradeController.py:1016
msgid "Could not download the upgrades"
msgstr "A frissítések nem tölthetők le"

#: ../DistUpgrade/DistUpgradeController.py:938
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"A frissítés most megszakadt. Ellenőrizze internetkapcsolatát vagy a "
"telepítőlemezt és próbálja újra. Minden eddig letöltött fájl megmarad."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:984
#: ../DistUpgrade/DistUpgradeController.py:1107
msgid "Error during commit"
msgstr "Hiba a módosítások rögzítése közben"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:986
#: ../DistUpgrade/DistUpgradeController.py:1146
msgid "Restoring original system state"
msgstr "A rendszer eredeti állapotának helyreállítása"

#: ../DistUpgrade/DistUpgradeController.py:987
#: ../DistUpgrade/DistUpgradeController.py:1000
msgid "Could not install the upgrades"
msgstr "A frissítések nem telepíthetők"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:991
msgid ""
"The upgrade is now aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"A frissítés most megszakadt. Előfordulhat, hogy a rendszer használhatatlan "
"állapotban van. Egy helyreállítás kerül futtatásra (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:996
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"A frissítés most megszakadt. Jelentse ezt a hibát az „update-manager” "
"csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban található "
"fájlokat a hibajelentésbe.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1017
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"A frissítés most megszakadt. Ellenőrizze internetkapcsolatát vagy a "
"telepítőlemezt és próbálja újra. "

#: ../DistUpgrade/DistUpgradeController.py:1095
msgid "Remove obsolete packages?"
msgstr "Eltávolítja az elavult csomagokat?"

#: ../DistUpgrade/DistUpgradeController.py:1096
#: ../DistUpgrade/DistUpgrade.glade.h:20
msgid "_Keep"
msgstr "_Megtartás"

#: ../DistUpgrade/DistUpgradeController.py:1096
msgid "_Remove"
msgstr "_Eltávolítás"

#: ../DistUpgrade/DistUpgradeController.py:1108
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"A frissítés befejező fázisa közben hiba lépett fel. Az alábbi üzenet további "
"információkat tartalmaz a hibára vonatkozóan. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1184
msgid "Required depends is not installed"
msgstr "A szükséges függőségek nincsenek telepítve"

#: ../DistUpgrade/DistUpgradeController.py:1185
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "A következő függőség nincs telepítve: „%s”. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1439
#: ../DistUpgrade/DistUpgradeController.py:1506
msgid "Checking package manager"
msgstr "Csomagkezelő ellenőrzése"

#: ../DistUpgrade/DistUpgradeController.py:1444
msgid "Preparing the upgrade failed"
msgstr "A frissítés előkészítése meghiúsult"

#: ../DistUpgrade/DistUpgradeController.py:1445
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"A rendszer frissítésre való előkészítése meghiúsult. Jelentse ezt a hibát az "
"„update-manager” csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban "
"található fájlokat a hibajelentésbe."

#: ../DistUpgrade/DistUpgradeController.py:1461
msgid "Getting upgrade prerequisites failed"
msgstr "A frissítés előfeltételeinek lekérése meghiúsult"

#: ../DistUpgrade/DistUpgradeController.py:1462
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"A rendszer nem volt képes lekérni a frissítés előfeltételeit. A frissítés "
"most megszakad és helyreáll a rendszer eredeti állapota.\n"
"\n"
"Jelentse ezt a hibát az „update-manager” csomaghoz és vegye be a "
"/var/log/dist-upgrade/ könyvtárban található fájlokat a hibajelentésbe."

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Updating repository information"
msgstr "Tárolóinformáció frissítése"

#: ../DistUpgrade/DistUpgradeController.py:1517
msgid "Invalid package information"
msgstr "Érvénytelen csomaginformációk"

#: ../DistUpgrade/DistUpgradeController.py:1518
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"A csomaginformációk frissítése után a(z) „%s” alapvető csomag már nem "
"található. Ez egy komoly problémát jelez, jelentse ezt a hibát az „update-"
"manager” csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban "
"található fájlokat a hibajelentésbe."

#: ../DistUpgrade/DistUpgradeController.py:1535
#: ../DistUpgrade/DistUpgradeController.py:1583
msgid "Fetching"
msgstr "Letöltés…"

#: ../DistUpgrade/DistUpgradeController.py:1542
#: ../DistUpgrade/DistUpgradeController.py:1587
msgid "Upgrading"
msgstr "Frissítés…"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1547
#: ../DistUpgrade/DistUpgradeController.py:1589
#: ../DistUpgrade/DistUpgradeController.py:1596
#: ../DistUpgrade/DistUpgradeController.py:1601
msgid "Upgrade complete"
msgstr "A frissítés befejeződött"

#: ../DistUpgrade/DistUpgradeController.py:1548
#: ../DistUpgrade/DistUpgradeController.py:1590
#: ../DistUpgrade/DistUpgradeController.py:1597
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr ""
"A frissítés befejeződött, de jelentkezett néhány hiba a folyamat során."

#: ../DistUpgrade/DistUpgradeController.py:1555
msgid "Searching for obsolete software"
msgstr "Elavult szoftverek keresése…"

#: ../DistUpgrade/DistUpgradeController.py:1560
msgid "System upgrade is complete."
msgstr "A rendszer frissítése befejeződött."

#: ../DistUpgrade/DistUpgradeController.py:1602
msgid "The partial upgrade was completed."
msgstr "A részleges frissítés sikeresen befejeződött."

#: ../DistUpgrade/DistUpgradeQuirks.py:166
#: ../DistUpgrade/DistUpgradeQuirks.py:235
#: ../DistUpgrade/DistUpgradeQuirks.py:260
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"A frissítés visszafoghatja a vizuális effekteket, a játékok és a sok "
"grafikai számítást igénylő alkalmazások teljesítményét."

#: ../DistUpgrade/DistUpgradeQuirks.py:170
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 9.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Az Ön gépében AMD 'fglrx' grafikus kártya van. Az Ubuntu 9.04 nem tartalmaz "
"olyan vezérlőprogramot, amely működne az Ön hardverével. \n"
"\n"
"Szeretné folytatni?"

#: ../DistUpgrade/DistUpgradeQuirks.py:239
#: ../DistUpgrade/DistUpgradeQuirks.py:264
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 8.10.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Ez a számítógép jelenleg az NVIDIA „nvidia” nevű eszközmeghajtóját "
"használja. Ehhez a videokártyához nem található az Ubuntu 8.10-zel működő "
"eszközmeghajtó.\n"
"\n"
"Folytatja?"

#: ../DistUpgrade/DistUpgradeQuirks.py:434
msgid "evms in use"
msgstr "Az evms használatban van"

#: ../DistUpgrade/DistUpgradeQuirks.py:435
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"A rendszer az „evms” kötetkezelőt használja a /proc/mounts alatt. Az „evms” "
"szoftver már nem támogatott, kapcsolja ki és futtassa újra a frissítést."

#: ../DistUpgrade/DistUpgradeMain.py:38
msgid "Sandbox upgrade using aufs"
msgstr "Tesztfrissítés aufs használatával"

#: ../DistUpgrade/DistUpgradeMain.py:40
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"A megadott útvonalat használja frissíthető csomagokat tartalmazó CD lemez "
"kereséséhez"

#: ../DistUpgrade/DistUpgradeMain.py:46
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Használja a kezelőfelületet. Jelenleg elérhető: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:49
msgid "*DEPRECATED* this option will be ignore"
msgstr "*ÉRVÉNYTELENÍTVE* ez az opció meg fog szűnni"

#: ../DistUpgrade/DistUpgradeMain.py:52
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Részleges frissítés végrehajtása (nem írja újra a sources.list nevű fájlt)"

#: ../DistUpgrade/DistUpgradeMain.py:54
msgid "Set datadir"
msgstr "Adatkönyvtár beállítása"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:108
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Helyezze be a(z) „%s” adathordozót a(z) „%s” meghajtóba."

#: ../DistUpgrade/DistUpgradeViewGtk.py:127
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "A letöltés befejeződött"

#: ../DistUpgrade/DistUpgradeViewGtk.py:139
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Fájl letöltése: %li / %li, (%sB/s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:140
#: ../DistUpgrade/DistUpgradeViewGtk.py:275
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:357
#, python-format
msgid "About %s remaining"
msgstr "Kb. %s van hátra"

#: ../DistUpgrade/DistUpgradeViewGtk.py:142
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "%li. fájl letöltése (összesen: %li)"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:174
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Módosítások alkalmazása…"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:199
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "függőségi hibák - e csomag beállítatlan maradt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:204
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "„%s” nem telepíthető"

#: ../DistUpgrade/DistUpgradeViewGtk.py:205
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may be in a not working "
"state. Please consider submitting a bug report about it."
msgstr ""
"A frissítés folytatódik, de a(z) „%s” csomag működésképtelen állapotban "
"lehet. Ezzel kapcsolatban küldjön be hibajelentést."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:221
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Lecseréli a személyre szabott\n"
"„%s”\n"
"konfigurációs fájlt?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:222
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"A beállítófájl összes módosítása elvész, ha lecseréli az újabb verzióra."

#: ../DistUpgrade/DistUpgradeViewGtk.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:309
msgid "The 'diff' command was not found"
msgstr "A „diff” parancs nem található"

#: ../DistUpgrade/DistUpgradeViewGtk.py:417
#: ../DistUpgrade/DistUpgradeViewText.py:107
msgid "A fatal error occurred"
msgstr "Fatális hiba jelentkezett!"

#: ../DistUpgrade/DistUpgradeViewGtk.py:418
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Jelentse ezt a hibát (ha eddig még nem tette volna) és csatolja a "
"/var/log/dist-upgrade/main.log és /var/log/dist-upgrade/apt.log nevű "
"fájlokat. A frissítés most megszakadt.\n"
"Az eredeti sources.list /etc/apt/sources.list.distUpgrade néven került "
"mentésre."

#: ../DistUpgrade/DistUpgradeViewGtk.py:433
msgid "Ctrl-c pressed"
msgstr "Ctrl+C billentyűkombináció lenyomva"

#: ../DistUpgrade/DistUpgradeViewGtk.py:434
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Ez megszakítja a műveletet és esetleg törött állapotban hagyja a rendszert. "
"Biztos, hogy ezt akarja tenni?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:570
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Az esetleges adatvesztés elkerülése érdekében zárjon be minden nyitott "
"alkalmazást és dokumentumot."

#: ../DistUpgrade/DistUpgradeViewGtk.py:584
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr "<b>%s visszafejlesztése</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:586
#: ../DistUpgrade/DistUpgradeViewKDE.py:750
#, python-format
msgid "Remove %s"
msgstr "%s eltávolítása"

#: ../DistUpgrade/DistUpgradeViewGtk.py:591
#: ../DistUpgrade/DistUpgradeViewKDE.py:752
#, python-format
msgid "Install %s"
msgstr "%s telepítése"

#: ../DistUpgrade/DistUpgradeViewGtk.py:593
#: ../DistUpgrade/DistUpgradeViewKDE.py:754
#, python-format
msgid "Upgrade %s"
msgstr "%s frissítése"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "Adathordozó-csere"

#: ../DistUpgrade/DistUpgradeViewKDE.py:321
msgid "Show Difference >>>"
msgstr "Különbség megjelenítése >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:324
msgid "<<< Hide Difference"
msgstr "<<< Különbség elrejtése"

#: ../DistUpgrade/DistUpgradeViewKDE.py:531
msgid "Error"
msgstr "Hiba"

#: ../DistUpgrade/DistUpgradeViewKDE.py:543
msgid "&Cancel"
msgstr "&Mégsem"

#: ../DistUpgrade/DistUpgradeViewKDE.py:545
#: ../DistUpgrade/DistUpgradeViewKDE.py:782
msgid "&Close"
msgstr "&Bezárás"

#: ../DistUpgrade/DistUpgradeViewKDE.py:591
msgid "Show Terminal >>>"
msgstr "Terminál megjelenítése >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:594
msgid "<<< Hide Terminal"
msgstr "<<< Terminál elrejtése"

#: ../DistUpgrade/DistUpgradeViewKDE.py:674
msgid "Information"
msgstr "Információ"

#: ../DistUpgrade/DistUpgradeViewKDE.py:724
#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#: ../DistUpgrade/DistUpgradeViewKDE.py:768
#: ../DistUpgrade/DistUpgrade.glade.h:9
msgid "Details"
msgstr "Részletek"

#: ../DistUpgrade/DistUpgradeViewKDE.py:778
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "Restart required"
msgstr "Újraindítás szükséges"

#: ../DistUpgrade/DistUpgradeViewKDE.py:778
#: ../DistUpgrade/DistUpgrade.glade.h:5
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Indítsa újra a rendszert a frissítés befejezéséhez</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgrade.glade.h:23
msgid "_Restart Now"
msgstr "Újrain_dítás most"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:793
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Megszakítja a frissítést?</big></b>\n"
"\n"
"Lehet, hogy a rendszer használhatatlanná válik, ha most megszakítja a "
"frissítést. Erősen ajánlott a frissítés folytatása!"

#: ../DistUpgrade/DistUpgradeViewKDE.py:797
msgid "Cancel Upgrade?"
msgstr "Megszakítja a frissítést?"

#: ../DistUpgrade/DistUpgradeView.py:59
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li nap"
msgstr[1] "%li nap"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li óra"
msgstr[1] "%li óra"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li perc"
msgstr[1] "%li perc"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li másodperc"
msgstr[1] "%li másodperc"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:77
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:92
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:113
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"A letöltés körülbelül %sig tart 1 megabites DSL kapcsolaton és %sig 56k-s "
"modem használatával."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:117
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "A letöltés körülbelül %sig tart ezen a kapcsolaton. "

#: ../DistUpgrade/DistUpgradeView.py:207 ../DistUpgrade/DistUpgrade.glade.h:14
msgid "Preparing to upgrade"
msgstr "Felkészülés a frissítésre"

#: ../DistUpgrade/DistUpgradeView.py:208
msgid "Getting new software channels"
msgstr "Új szoftvercsatornák lekérése"

#: ../DistUpgrade/DistUpgradeView.py:209 ../DistUpgrade/DistUpgrade.glade.h:12
msgid "Getting new packages"
msgstr "Új csomagok letöltése"

#: ../DistUpgrade/DistUpgradeView.py:210 ../DistUpgrade/DistUpgrade.glade.h:13
msgid "Installing the upgrades"
msgstr "A frissítések telepítése"

#: ../DistUpgrade/DistUpgradeView.py:211 ../DistUpgrade/DistUpgrade.glade.h:8
msgid "Cleaning up"
msgstr "Frissítés befejezése"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:284
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d csomag el lesz távolítva."
msgstr[1] "%d csomag el lesz távolítva."

#: ../DistUpgrade/DistUpgradeView.py:289
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d új csomag lesz telepítve."
msgstr[1] "%d új csomag lesz telepítve."

#: ../DistUpgrade/DistUpgradeView.py:295
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d csomag lesz frissítve."
msgstr[1] "%d csomag lesz frissítve."

#: ../DistUpgrade/DistUpgradeView.py:300
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Összes letöltendő adatmennyiség: %s. "

#: ../DistUpgrade/DistUpgradeView.py:304
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"A frissítés letöltése és telepítése több órát is igénybe vehet és a letöltés "
"befejeződése után már nem szakítható meg."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:310 ../UpdateManager/UpdateManager.py:456
msgid "Your system is up-to-date"
msgstr "A rendszere naprakész"

#: ../DistUpgrade/DistUpgradeView.py:311
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Nem állnak rendelkezésre frissítések a rendszeréhez. A frissítés most "
"megszakad."

#: ../DistUpgrade/DistUpgradeView.py:324
msgid "Reboot required"
msgstr "Újraindítás szükséges"

#: ../DistUpgrade/DistUpgradeView.py:325
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"A frissítés elkészült, de a befejezéshez újra kell indítani a rendszert. "
"Újraindítja most?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:134
#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:134
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid "Could not run the upgrade tool"
msgstr "Nem sikerült futtatni a frissítőeszközt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Feltehetőleg ez egy hiba a frissítőeszközben. Kérem, jelentse ezt hibaként."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool signature"
msgstr "Frissítőeszköz aláírása"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool"
msgstr "Frissítőeszköz"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "A letöltés meghiúsult"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"A frissítés letöltése meghiúsult. Lehetséges, hogy hálózati probléma áll "
"fenn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "A kibontás meghiúsult"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"A frissítés kibontása meghiúsult. Probléma lehet a hálózattal vagy a "
"kiszolgálóval. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verfication failed"
msgstr "Az ellenőrzés meghiúsult"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"A frissítés ellenőrzése meghiúsult. Probléma lehet a hálózattal vagy a "
"kiszolgálóval. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "Hitelesítés sikertelen"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"A frissítés hitelesítése meghiúsult. Probléma lehet a hálózattal vagy a "
"kiszolgálóval. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:263
#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:263
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Can not run the upgrade"
msgstr "A frissítés nem futtatható!"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:264
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:264
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Ez általában olyan rendszer esetén jelentkezik, melyben a /tmp kötetet "
"noexec opcióval csatolták. Kérem csatolja le a kötetet a noexec opció "
"nélkül, majd indítsa el ismét a frissítőt."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
#, python-format
msgid "The error message is '%s'."
msgstr "A hibaüzenet: „%s”."

#: ../DistUpgrade/DistUpgradeViewText.py:108
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Jelentse ezt hibaként és a hibajelentéshez mellékelje a /var/log/dist-"
"upgrade/main.log és /var/log/dist-upgrade/apt.log fájlokat. A frissítés most "
"megszakadt.\n"
"Az eredeti sources.list /etc/apt/sources.list.distUpgrade néven került "
"mentésre."

#: ../DistUpgrade/DistUpgradeViewText.py:132
msgid "Aborting"
msgstr "Megszakítás…"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "Demoted:\n"
msgstr "Lefokozott:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
#: ../DistUpgrade/DistUpgradeViewText.py:198
msgid "Continue [yN] "
msgstr "Folytatja? [i/N] "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
msgid "Details [d]"
msgstr "Részletek [r]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:173
#: ../DistUpgrade/DistUpgradeViewText.py:201
msgid "y"
msgstr "i"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:176
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:179
msgid "d"
msgstr "r"

#: ../DistUpgrade/DistUpgradeViewText.py:183
#, python-format
msgid "Remove: %s\n"
msgstr "Eltávolítandó: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
#, python-format
msgid "Install: %s\n"
msgstr "Telepítendő: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:189
#, python-format
msgid "Upgrade: %s\n"
msgstr "Frissítendő: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
msgid "Continue [Yn] "
msgstr "Folytatja? [Y - igen / n - nem] "

#: ../DistUpgrade/DistUpgradeViewText.py:226
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"A frissítés befejezéséhez a rendszer újraindítása szükséges. \n"
"Az „i” választása esetén a rendszer újraindul."

#: ../DistUpgrade/DistUpgrade.glade.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.glade.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Megszakítja a folyamatban lévő frissítést?</big></b>\n"
"\n"
"A rendszer használhatatlan állapotban maradhat, ha megszakítja a frissítést. "
"Erősen javasoljuk, hogy folytassa a frissítést."

#: ../DistUpgrade/DistUpgrade.glade.h:6
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Megkezdi a frissítést?</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:7
msgid "<b><big>Upgrading Ubuntu to version 9.04</big></b>"
msgstr "<b><big>Frissítés Ubuntu 9.04-re</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:10
msgid "Difference between the files"
msgstr "Fájlok közti különbség"

#: ../DistUpgrade/DistUpgrade.glade.h:11
msgid "Distribution Upgrade"
msgstr "Disztribúciófrissítés"

#: ../DistUpgrade/DistUpgrade.glade.h:15
msgid "Restarting the computer"
msgstr "A számítógép újraindítása"

#: ../DistUpgrade/DistUpgrade.glade.h:16
msgid "Setting new software channels"
msgstr "Új szoftvercsatornák beállítása"

#: ../DistUpgrade/DistUpgrade.glade.h:17
msgid "Terminal"
msgstr "Terminál"

#: ../DistUpgrade/DistUpgrade.glade.h:18
msgid "_Cancel Upgrade"
msgstr "Frissítés _megszakítása"

#: ../DistUpgrade/DistUpgrade.glade.h:19
msgid "_Continue"
msgstr "_Folytatás"

#: ../DistUpgrade/DistUpgrade.glade.h:21
msgid "_Replace"
msgstr "_Csere"

#: ../DistUpgrade/DistUpgrade.glade.h:22
msgid "_Report Bug"
msgstr "_Hibabejelentés"

#: ../DistUpgrade/DistUpgrade.glade.h:24
msgid "_Resume Upgrade"
msgstr "Frissítés _folytatása"

#: ../DistUpgrade/DistUpgrade.glade.h:25
msgid "_Start Upgrade"
msgstr "Frissítés _indítása"

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "Nem érhetők el a kiadási megjegyzések"

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "A kiszolgáló túl lehet terhelve. "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "A kiadási megjegyzések nem tölthetők le"

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "Kérjük ellenőrizze az internetkapcsolatát."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "Frissítés"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.glade.h:11
msgid "Release Notes"
msgstr "Kiadási megjegyzések"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "További csomagfájlok letöltése..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "%s. fájl, összesen %s. Sebesség: %sB/mp"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "%s. fájl, összesen %s."

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "Hivatkozás megnyitása a böngészőben"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "Hivatkozás másolása a vágólapra"

#: ../UpdateManager/GtkProgress.py:111
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"%(current)li. fájl letöltése, összesen: %(total)li, sebesség: %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:116
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "%(current)li. fájl letöltése, összesen: %(total)li"

#: ../UpdateManager/UpdateManager.py:274
#, python-format
msgid "Version %s: \n"
msgstr "%s verzió: \n"

#: ../UpdateManager/UpdateManager.py:335
msgid "Downloading list of changes..."
msgstr "Változások listájának letöltése…"

#: ../UpdateManager/UpdateManager.py:367
msgid "_Uncheck All"
msgstr "Kijelölések _törlése"

#: ../UpdateManager/UpdateManager.py:373
msgid "_Check All"
msgstr "Összes _kijelölése"

#. TRANSLATORS: b stands for Bytes
#: ../UpdateManager/UpdateManager.py:417 ../UpdateManager/UpdateManager.py:477
#, python-format
msgid "Download size: %s"
msgstr "Letöltés mérete: %s"

#: ../UpdateManager/UpdateManager.py:421
msgid "Unknown download size"
msgstr "A letöltés mérete ismeretlen"

#: ../UpdateManager/UpdateManager.py:429
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr ""
"Nem ismert, hogy mikor történt a csomaginformációk utolsó frissítése. Az "
"„Ellenőrzés” gombra kattintva frissítheti az információt."

#: ../UpdateManager/UpdateManager.py:438
#, python-format
msgid "The package information was last updated %s day ago."
msgid_plural "The package information was last updated %s days ago."
msgstr[0] "A csomaginformációk %s nappal ezelőtt frissültek."
msgstr[1] "A csomaginformációk %s nappal ezelőtt frissültek."

#: ../UpdateManager/UpdateManager.py:442
#, python-format
msgid "The package information was last updated %s hour ago."
msgid_plural "The package information was last updated %s hours ago."
msgstr[0] "A csomaginformációk %s órája frissültek."
msgstr[1] "A csomaginformációk %s órája frissültek."

#: ../UpdateManager/UpdateManager.py:446
msgid "The package information was last updated less than one hour ago."
msgstr "A csomaginformációk kevesebb mint 1 órája frissültek."

#: ../UpdateManager/UpdateManager.py:454
#: ../data/glade/UpdateManager.glade.h:21
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"A szoftverfrissítések kijavítják a programhibákat és biztonsági "
"sebezhetőségeket, illetve új szolgáltatásokat biztosítanak."

#: ../UpdateManager/UpdateManager.py:471
msgid "Welcome to Ubuntu"
msgstr "Üdvözli az Ubuntu"

#: ../UpdateManager/UpdateManager.py:472
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"Ezen szoftverfrissítések az Ubuntu kiadása után jelentek meg. Ha ezeket most "
"nem kívánja telepíteni, akkor a későbbiekben válassza a Rendszer → "
"Adminisztráció menü Frissítéskezelő menüpontját."

#: ../UpdateManager/UpdateManager.py:475
msgid "Software updates are available for this computer"
msgstr "Szoftverfrissítések érhetőek el ehhez a számítógéphez"

#: ../UpdateManager/UpdateManager.py:476
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Ha ezeket most nem kívánja telepíteni, akkor a későbbiekben válassza a "
"Rendszer → Adminisztráció menü Frissítéskezelő menüpontját."

#: ../UpdateManager/UpdateManager.py:515
msgid "Please wait, this can take some time."
msgstr "Kis türelmet, ez eltarthat egy ideig."

#: ../UpdateManager/UpdateManager.py:517
msgid "Update is complete"
msgstr "A frissítés befejeződött"

#: ../UpdateManager/UpdateManager.py:564
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"A frissítéshez %s szabad helyre van szükség a '%s' lemezen. Kérem "
"szabadítson fel még legalább %s helyet a '%s' lemezen. Ürítse ki a "
"szemetesét és távolítsa el a korábbi telepítések ideiglenes csomagjait  a "
"'sudo apt-get clean' parancs használatával."

#: ../UpdateManager/UpdateManager.py:603
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "Csomaginformációk olvasása…"

#: ../UpdateManager/UpdateManager.py:705
msgid "Could not initialize the package information"
msgstr "A csomaginformációk inicializálása meghiúsult"

#: ../UpdateManager/UpdateManager.py:706
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Megoldhatatlan probléma lépett fel a csomaginformációk inicializálása "
"közben.\n"
"\n"
"Kérem jelentse ezt az „update-manager” csomag hibájaként és csatolja a "
"következő hibaüzenetet:\n"

#: ../UpdateManager/UpdateManager.py:728
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Megoldhatatlan probléma lépett fel a frissítés alatt.\n"
"\n"
"Kérem jelentse ezt az „update-manager” csomag hibájaként és csatolja a "
"következő hibaüzenetet:"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:753
#, python-format
msgid "(Size: %s)"
msgstr "(Méret: %s)"

#: ../UpdateManager/UpdateManager.py:755
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Régi verzió: %(old_version)s, új verzió:  %(new_version)s"

#: ../UpdateManager/UpdateManager.py:759
#, python-format
msgid "Version %s"
msgstr "%s verzió"

#: ../UpdateManager/UpdateManager.py:772
msgid "Your distribution is not supported anymore"
msgstr "A terjesztés már nem támogatott"

#: ../UpdateManager/UpdateManager.py:773
msgid ""
"You will not get any further security fixes or critical updates. Upgrade to "
"a later version of Ubuntu Linux. See http://www.ubuntu.com for more "
"information on upgrading."
msgstr ""
"Nem fog további biztonsági javításokat vagy kritikus frissítéseket kapni. "
"Frissítsen az Ubuntu Linux egy újabb változatára. A frissítéssel kapcsolatos "
"információkat az http://www.ubuntu.com weboldalon talál."

#: ../UpdateManager/UpdateManager.py:805
#, python-format
msgid "<b>New distribution release '%s' is available</b>"
msgstr "<b>A disztribúció új „%s” kiadása elérhető</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:844
msgid "Software index is broken"
msgstr "A szoftverindex sérült"

#: ../UpdateManager/UpdateManager.py:845
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Lehetetlen szoftvereket telepíteni vagy törölni. Használja a Synaptic "
"csomagkezelőt vagy futtassa a „sudo apt-get install -f” parancsot egy "
"terminálban a probléma megoldása érdekében."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "Csomaginformációk letöltése"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr ""
"Új, eltávolított vagy frissített szoftvercsomagok keresése a tárolókban"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "Csomagfrissítések letöltése"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr ""
"A kiválasztott csomagfrissítések letöltése és telepítése folyamatban..."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr "A run_synaptic-ot ismeretlen parancs hívta meg"

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "Mégsem"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "Telepít"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "Frissítésilista létrehozása"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" ◦ A previous upgrade which didn't complete\n"
" ◦ Problems with some of the installed software\n"
" ◦ Unofficial software packages not provided by Ubuntu\n"
" ◦ Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Nem tudtam előkészíteni hagyományos frissítést. Kérem futassa parancssorban "
"a következőt: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"A fenti problémát okozhatta:\n"
" ◦ egy folyamatban lévő frissítés\n"
" ◦ problémák egy telepített programmal\n"
" ◦ nemhivatalos csomagok, amiket nem az Ubuntu biztosít\n"
" ◦ szokásos változások egy kiadás előtti Ubuntu verzióban."

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "Változások listájának letöltése"

#. FIXME: better string, but we are in string freeze
#: ../UpdateManager/Core/MyCache.py:221 ../UpdateManager/Core/UpdateList.py:47
msgid "Failed to detect distribution"
msgstr "Hiba a kiadás lekérdezése közben"

#: ../UpdateManager/Core/MyCache.py:228
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"A változások listája nem tartalmaz semmilyen lényeges változást.\n"
"\n"
"Kérem használja a http://launchpad.net/ubuntu/+source/%s/%s/+changelog \n"
"oldalt, amíg a változtatások elérhetővé nem válnak vagy próbálja újra később."

#: ../UpdateManager/Core/MyCache.py:233
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"A változtatások listája még nem érhető el.\n"
"\n"
"Addig látogasson el a http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"oldalra, míg nem válik hozzáférhetővé, vagy próbálja meg később!"

#: ../UpdateManager/Core/MyCache.py:239
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"A módosítások listájának letöltése meghiúsult.\n"
"Ellenőrizze az internetkapcsolatát."

#: ../UpdateManager/Core/UpdateList.py:48
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "A következő hiba lépett fel a rendszer felmérése során: „%s”."

#: ../UpdateManager/Core/UpdateList.py:59
msgid "Important security updates"
msgstr "Fontos biztonsági frissítések"

#: ../UpdateManager/Core/UpdateList.py:60
msgid "Recommended updates"
msgstr "Ajánlott frissítések"

#: ../UpdateManager/Core/UpdateList.py:61
msgid "Proposed updates"
msgstr "Javasolt frissítések"

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Backports"
msgstr "Visszaportolt csomagok"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Distribution updates"
msgstr "Disztribúciófrissítések"

#: ../UpdateManager/Core/UpdateList.py:68
msgid "Other updates"
msgstr "Egyéb frissítések"

#: ../UpdateManager/Core/UpdateList.py:77
msgid "Previous selected"
msgstr "Előző kijelölés"

#: ../data/glade/UpdateManager.glade.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Saját kezűleg kell frissítenie a rendszert</big></b>\n"
"\n"
"A rendszere jelenleg nem keresi automatikusan a frissítéseket. Ezt a "
"viselkedést a <i>Frissítések</i> lap <i>Szoftverforrások</i> részében "
"változtathatja meg."

#: ../data/glade/UpdateManager.glade.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Nem minden frissítés telepíthető</b></big>"

#: ../data/glade/UpdateManager.glade.h:5
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Frissítéskezelő indítása</b></big>"

#: ../data/glade/UpdateManager.glade.h:6
msgid "Changes"
msgstr "Módosítások"

#: ../data/glade/UpdateManager.glade.h:7
msgid "Chec_k"
msgstr "_Ellenőrzés"

#: ../data/glade/UpdateManager.glade.h:8
msgid "Check the software channels for new updates"
msgstr "Frissítések keresése a szoftvercsatornákon"

#: ../data/glade/UpdateManager.glade.h:9
msgid "Description"
msgstr "Leírás"

#: ../data/glade/UpdateManager.glade.h:10
msgid "Description of update"
msgstr "Frissítés leírása"

#: ../data/glade/UpdateManager.glade.h:12
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" ◦ A previous upgrade which didn't complete\n"
" ◦ Problems with some of the installed software\n"
" ◦ Unofficial software packages not provided by Ubuntu\n"
" ◦ Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Részleges frissítés futtatása, a lehető legtöbb frissítés telepítése "
"érdekében. \n"
"\n"
"Ezt\n"
" ◦ egy korábbi befejezetlen frissítés;\n"
" ◦ egyes telepített szoftverekkel kapcsolatos problémák;\n"
" ◦ nem hivatalos, nem az Ubuntu által szállított szoftvercsomagok vagy\n"
" ◦ az Ubuntu fejlesztői verziójának normális változásai okozhatják."

#: ../data/glade/UpdateManager.glade.h:19
msgid "Settings…"
msgstr "Beállítások..."

#: ../data/glade/UpdateManager.glade.h:20
msgid "Show progress of individual files"
msgstr "Egyes fájlok állapotának mutatása"

#: ../data/glade/UpdateManager.glade.h:22
msgid "U_pgrade"
msgstr "_Frissítés"

#: ../data/glade/UpdateManager.glade.h:23
#: ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "Frissítéskezelő"

#: ../data/glade/UpdateManager.glade.h:24
msgid "Upgrade to the latest version of Ubuntu"
msgstr "Frissítés az Ubuntu legújabb változatára"

#: ../data/glade/UpdateManager.glade.h:25
msgid "_Hide this information in the future"
msgstr "_A jövőben ne mutassa ezt az információt"

#: ../data/glade/UpdateManager.glade.h:26
msgid "_Install Updates"
msgstr "_Telepítés"

#: ../data/glade/UpdateManager.glade.h:27
msgid "_Partial Upgrade"
msgstr "_Részleges frissítés"

#: ../data/glade/UpdateManager.glade.h:28
msgid "_Upgrade"
msgstr "_Frissítés"

#: ../data/glade/UpdateManager.glade.h:29
msgid "updates"
msgstr "frissítés"

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Rendelkezésre álló frissítések megjelenítése és telepítése"

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "Szoftverfrissítések"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "Új disztribúciókiadások keresése (elavult)"

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr "A telepítőablak automatikus bezárása sikeres telepítés esetén"

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr "Üdvözlőszöveg (csak az első futáskor jelenik meg)"

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"Ha a frissítések automatikus keresése le van tiltva, akkor a csatornalistát "
"kézzel kell újratölteni. Ezzel a beállítással elrejtheti az ilyen "
"helyzetekben megjelenő emlékeztetőt."

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr "E kulcs kiválasztásával az üdvözlőszöveg megjeleníthető."

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr ""
"Ha ez az opció be van állítva, a telepítőablak automatikusan bezáródik "
"sikeres telepítés esetén."

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr ""
"Ha ez a kulcs be van állítva, a fő frissítési ablakban meg fog jelenni a "
"verzióinformáció (hogy melyik verzióról melyikre frissít)."

#: ../data/update-manager.schemas.in.h:8
msgid "Remind to reload the channel list"
msgstr "Figyelmeztessen, ha újra kell tölteni a csatornalistát"

#: ../data/update-manager.schemas.in.h:9
msgid "Show details of an update"
msgstr "Frissítés részleteinek megjelenítése"

#: ../data/update-manager.schemas.in.h:10
msgid "Show version in update list"
msgstr "Verzió megjelenítése a frissítési listában"

#: ../data/update-manager.schemas.in.h:11
msgid "Stores the size of the update-manager dialog"
msgstr "A frissítéskezelő ablak méretének tárolása"

#: ../data/update-manager.schemas.in.h:12
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""
"Eltárolja a módosítások listáját és a leírást tartalmazó kiterjesztő "
"állapotát"

#: ../data/update-manager.schemas.in.h:13
msgid "The window size"
msgstr "Az ablak mérete"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"Ez a kulcs elavult, helyette az /etc/update-manager/release-upgrades fájl "
"használatos. Az aktuális terjesztés új verziójának automatikus keresése és a "
"frissítés felajánlása (ha lehetséges)."

#: ../update-manager:55
msgid "Show version and exit"
msgstr "Verziószám kiírása és kilépés"

#: ../update-manager:58
msgid "Check if a new distribution release is available"
msgstr "Új disztribúciókiadások keresése"

#: ../update-manager:61 ../do-release-upgrade:23 ../check-new-release:24
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Ellenőrzi, hogy lehetséges-e frissíteni a legújabb fejlesztői kiadásra"

#: ../update-manager:65
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Frissítés a legújabb verziófrissítő segítségével"

#: ../update-manager:72
msgid "Do not focus on map when starting"
msgstr "Ne fókuszáljon a térképre indításkor!"

#: ../update-manager:75
msgid "Try to run a dist-upgrade"
msgstr "Megpróbál disztribúciófrissítést futtatni"

#: ../update-manager:79 ../do-release-upgrade:39
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Tesztfrissítés tesztkörnyezetben aufs használatával"

#: ../update-manager:98
msgid "Running partial upgrade"
msgstr "Részleges frissítés futtatása"

#: ../do-release-upgrade:27 ../check-new-release:28
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Kísérlet a legfrissebb kiadásra való áttérésre $distro-proposed kiadásról a "
"frissítő használatával"

#: ../do-release-upgrade:31
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Futtatás speciális frissítő módban.\n"
"Jelenleg a „desktop” (asztali rendszerek szabályos frissítése) és a „server” "
"(kiszolgálórendszerek frissítése) módok támogatottak."

#: ../do-release-upgrade:37
msgid "Run the specified frontend"
msgstr "A megadott előtét futtatása"

#: ../do-release-upgrade:43
msgid "Checking for a new ubuntu release"
msgstr "Új Ubuntu-kiadás keresése"

#: ../do-release-upgrade:50
msgid "No new release found"
msgstr "Nem található új kiadás"

#: ../check-new-release:32
#, c-format
msgid ""
"Do not output anything, just return %s if there is a new release and %s if "
"there is none."
msgstr ""
"Ha új kiadás érhető el, jelenítse meg a következőt: %s, ellenkező esetben "
"pedig ezt: %s."

#: ../check-new-release:46
#, c-format
msgid "New release '%s' available."
msgstr "Elérhető az új „%s” kiadás."

#: ../check-new-release:47
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "A frissítéshez futassa a 'do-release-upgrade' parancsot."

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Megvalósítatlan módszer: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Egy állomány a lemezen"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb csomag"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Hiányzó csomag telepítése."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "A(z) %s csomagot telepíteni kell."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i elavult bejegyzés van az állapotfájlban"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Elavult bejegyzések a dpkg státuszban"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Elavult dpkg státuszbejegyzések"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug ##279621 for details."
msgstr ""
"Frissítéskor, ha a kdelibs4-dev telepítve van, a kdelibs5-dev csomagot is "
"telepíteni kell. Részletekért lásd a bugs.launchpad.net #279621 számú "
"hibáját."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "A(z) %s csomagot saját kezűleg telepítettként kell megjelölni."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"A lilo eltávolítása, mivel a grub is telepítve van. (Lásd a #314004 számú "
"hibát a részletekért)."

#~ msgid ""
#~ "If none of this applies, then please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the "
#~ "bugreport."
#~ msgstr ""
#~ "Ha ezek közül egyik sem teljesül, jelentse ezt az „update-manager” csomaggal "
#~ "kapcsolatos hibát és mellékelje a /var/log/dist-upgrade/ könyvtárban "
#~ "található fájlokat a hibajelentéshez."

#~ msgid "The upgrade tool will guide you through the upgrade process."
#~ msgstr "A frissítőeszköz végigvezeti Önt a frissítési folyamaton."

#~ msgid "Can't upgrade required meta-packages"
#~ msgstr "A szükséges meta-csomagok nem frissíthetőek"

#~ msgid "A essential package would have to be removed"
#~ msgstr "Egy alapvető csomag eltávolításra kerülne"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Megoldhatatlan probléma jelentkezett a frissítés közben.\n"
#~ "\n"
#~ " Ezt a következők okozhatták:\n"
#~ " * Frissítés az Ubuntu egy kiadás előtti verziójára\n"
#~ " * Az Ubuntu egy kiadás előtti verziójának futtatása\n"
#~ " * Nemhivatalos csomagok, melyeket nem az Ubuntu szolgáltat\n"
#~ "\n"

#, python-format
#~ msgid ""
#~ "The upgrade aborts now. The upgrade needs a total of %s free space on disk "
#~ "'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
#~ "your trash and remove temporary packages of former installations using 'sudo "
#~ "apt-get clean'."
#~ msgstr ""
#~ "A frissítés most meg fog szakadni. A frissítéshez %s szabad hely szükséges "
#~ "a(z) „%s” lemezen. Szabadítson fel további %s lemezterületet a(z) „%s” "
#~ "eszközön. Ürítse a kukát és törölje a korábbi telepítések átmeneti fájljait "
#~ "a „sudo apt-get clean” parancs kiadásával."

#~ msgid ""
#~ "The upgrade aborts now. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "A frissítés befejeződik! Kérem ellenőrizze internetkapcsolatát vagy a "
#~ "telepítőlemezt és próbálja újra. Minden eddigi letöltött fájl megmarad."

#~ msgid ""
#~ "The upgrade aborts now. Your system could be in an unusable state. A "
#~ "recovery will run now (dpkg --configure -a)."
#~ msgstr ""
#~ "A frissítés most félbeszakad. Előfordulhat, hogy a rendszer használhatatlan "
#~ "állapotban van. Egy helyreállítás kerül futtatásra (dpkg --configure -a)."

#, python-format
#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bugreport.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "A frissítés most félbeszakad. Jelentse ezt a hibát az „update-manager” "
#~ "csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban található "
#~ "fájlokat a hibajelentésbe.\n"
#~ "%s"

#~ msgid ""
#~ "The upgrade aborts now. Please check your Internet connection or "
#~ "installation media and try again. "
#~ msgstr ""
#~ "A frissítés megszakadt! Kérem ellenőrizze internetkapcsolatát vagy a "
#~ "telepítőlemezt és próbálja újra. "

#~ msgid ""
#~ "The upgrade is completed but there were errors during the ugprade process."
#~ msgstr "A frissítés befejeződött, de a folyamat során hibák léptek fel."

#~ msgid ""
#~ "Please report this as a bug (if you haven't already) and include the files "
#~ "/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
#~ "report. The upgrade aborts now.\n"
#~ "Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
#~ msgstr ""
#~ "Kérem jelentse a hibát (ha eddig még nem tette volna) és csatolja a "
#~ "/var/log/dist-upgrade/main.log és /var/log/dist-upgrade/apt.log nevű "
#~ "fájlokat. A frissítés megszakadt!"

#, python-format
#~ msgid "%(str_days)s %(str_hours)s %(str_minutes)s"
#~ msgstr "%(str_days)s %(str_hours)s %(str_minutes)s"

#, python-format
#~ msgid ""
#~ "This download will take about %s with a 1Mbit DSL connection and about %s "
#~ "with a 56k modem"
#~ msgstr ""
#~ "A letöltés körülbelül %sig tart 1 megabites DSL kapcsolaton és %sig 56k-s "
#~ "modem használatával"

#~ msgid ""
#~ "Please report this as a bug and include the files /var/log/dist-"
#~ "upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
#~ "upgrade aborts now.\n"
#~ "Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
#~ msgstr ""
#~ "Jelentse ezt hibaként és a hibajelentéshez mellékelje a /var/log/dist-"
#~ "upgrade/main.log és /var/log/dist-upgrade/apt.log fájlokat. A frissítés most "
#~ "félbeszakad.\n"
#~ "Az eredeti sources.list /etc/apt/sources.list.distUpgrade néven került "
#~ "mentésre."

#~ msgid "Restart the system now [yN] "
#~ msgstr "A rendszer újraindítása most [i/N] "

#~ msgid "<b><big>Upgrading Ubuntu to version 8.10</big></b>"
#~ msgstr "<b><big>Ubuntu frissítése a 8.10-es verzióra</big></b>"

#~ msgid "The list of changes is not available"
#~ msgstr "A módosítások listája nem érhető el"

#, python-format
#~ msgid "You can install %s update."
#~ msgid_plural "You can install %s updates."
#~ msgstr[0] "%s frissítést telepíthet,"
#~ msgstr[1] "%s frissítést telepíthet."

#~ msgid "<big><b>Keep your system up-to-date</b></big>"
#~ msgstr "<big><b>Tartsa naprakészen a rendszerét</b></big>"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this as a bug "
#~ "against the 'update-manager' package and include the files in /var/log/dist-"
#~ "upgrade/ in the bugreport."
#~ msgstr ""
#~ "A rendszer frissítésre való előkészítése meghiúsult. Jelentse ezt a hibát az "
#~ "„update-manager” csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban "
#~ "található fájlokat a hibajelentésbe."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The upgrade "
#~ "will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug against the 'update-manager' package and include "
#~ "the files in /var/log/dist-upgrade/ in the bugreport."
#~ msgstr ""
#~ "A rendszer nem volt képes lekérni a frissítés előfeltételeit. A frissítés "
#~ "most megszakad és helyreáll a rendszer eredeti állapota.\n"
#~ "\n"
#~ "Jelentse ezt a hibát az „update-manager” csomaghoz és vegye be a "
#~ "/var/log/dist-upgrade/ könyvtárban található fájlokat a hibajelentésbe."

#, python-format
#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug against the 'update-"
#~ "manager' package and include the files in /var/log/dist-upgrade/ in the "
#~ "bugreport."
#~ msgstr ""
#~ "A csomaginformációk frissítése után a(z) „%s” alapvető csomag már nem "
#~ "található.\n"
#~ "Ez egy komoly problémát jelez, jelentse ezt a hibát az „update-manager” "
#~ "csomaghoz és vegye be a /var/log/dist-upgrade/ könyvtárban található "
#~ "fájlokat a hibajelentésbe."

#, python-format
#~ msgid ""
#~ "The upgrade will continue but the '%s' package may be in a not working "
#~ "state. Please consider submitting a bugreport about it."
#~ msgstr ""
#~ "A frissítés folytatódik, de a(z) %s lehet, hogy nem fog működni. Ebben az "
#~ "esetben kérjük, jelentse be a hibát."

#~ msgid "Downloading the upgrade tool"
#~ msgstr "Frissítőeszköz letöltése"

#~ msgid "Daily"
#~ msgstr "Naponta"

#~ msgid "Every two days"
#~ msgstr "Kétnaponta"

#~ msgid "Weekly"
#~ msgstr "Hetente"

#~ msgid "Every two weeks"
#~ msgstr "Minden két hétben"

#~ msgid "After one week"
#~ msgstr "Egy hét után"

#~ msgid "After two weeks"
#~ msgstr "Két hét után"

#~ msgid "After one month"
#~ msgstr "Egy hónap után"

#~ msgid "Fetching file %li of %li at %sb/s"
#~ msgstr "Fájl letöltése (%li., összesen: %li), sebesség: %s b/s"

#~ msgid "%li days %li hours %li minutes"
#~ msgstr "%li nap, %li óra és %li perc"

#~ msgid "_Check"
#~ msgstr "_Ellenőrzés"

#~ msgid "Import key"
#~ msgstr "Kulcs importálása"

#~ msgid "Error importing selected file"
#~ msgstr "Hiba a kiválasztott fájl importálása közben"

#~ msgid "Error removing the key"
#~ msgstr "Hiba a kulcs eltávolítása közben"

#~ msgid "Checking for updates"
#~ msgstr "Frissítések keresése"

#~ msgid "Every %s days"
#~ msgstr "%s naponta"

#~ msgid "After %s days"
#~ msgstr "%s nap után"

#~ msgid "Please enter a name for the disc"
#~ msgstr "Kérem, adja meg a lemez nevét"

#~ msgid "Please insert a disc in the drive:"
#~ msgstr "Kérem, helyezzen be egy lemezt a meghajtóba:"

#~ msgid "Authentication"
#~ msgstr "Hitelesítés"

#~ msgid "_Check for updates automatically:"
#~ msgstr "_Frissítések automatikus keresése:"

#~ msgid "_Import Key File"
#~ msgstr "_Kulcsfájl importálása"

#~ msgid "<b>Comment:</b>"
#~ msgstr "<b>Megjegyzés:</b>"

#~ msgid ""
#~ "Binary\n"
#~ "Source"
#~ msgstr ""
#~ "Bináris\n"
#~ "Forrás"

#~ msgid "Scanning CD-ROM"
#~ msgstr "CD-ROM átvizsgálása"

#~ msgid "_Reload"
#~ msgstr "F_rissítés"

#~ msgid "Officially supported"
#~ msgstr "Hivatalosan támogatott"

#~ msgid "Restricted copyright"
#~ msgstr "Szerzői jogi korlátozás alatt"

#~ msgid "     "
#~ msgstr "     "

#~ msgid ""
#~ "Use special upgrade mode. Available:\n"
#~ "desktop, server"
#~ msgstr ""
#~ "Használjon speciális frissítési módot. Elérhető:\n"
#~ "asztali, kiszolgálói"

#~ msgid ""
#~ "The list of changes is not available yet.\n"
#~ "Please try again later."
#~ msgstr ""
#~ "A módosítások listája még nem érhető el.\n"
#~ "Próbálkozzon később."

#~ msgid "_Skip This Step"
#~ msgstr "Ezen lé_pés kihagyása"

#~ msgid "Asking for confirmation"
#~ msgstr "Megerősítés kérése"

#~ msgid "<b>Remove %s</b>"
#~ msgstr "<b>%s eltávolítása</b>"

#~ msgid "Fetch data from the network for the upgrade?"
#~ msgstr "Letölt adatokat a hálózatról a frissítéshez?"

#~ msgid ""
#~ "The key you selected could not be removed. Please report this as a bug."
#~ msgstr ""
#~ "Az Ön által kijelölt kulcs nem távolítható el. Kérem jelentse ezt hibaként."

#~ msgid "<p>You have to download a total of %s. "
#~ msgstr "<p>Összes letöltendő adatmennyiség: %s. "

#~ msgid "The selected file may not be a GPG key file or it might be corrupt."
#~ msgstr "A kiválasztott fájl vagy nem GPG kulcsfájl, vagy sérült."

#~ msgid "<b>Internet updates</b>"
#~ msgstr "<b>Hálózati frissítések</b>"

#~ msgid "Internet Updates"
#~ msgstr "Hálózati frissítések"

#~ msgid "<b>URI:</b>"
#~ msgstr "<b>URI:</b>"

#~ msgid "System Update Tool"
#~ msgstr "Rendszerfrissítő eszköz"

#~ msgid "Fetching the upgrades"
#~ msgstr "A frissítések letöltése"

#~ msgid ""
#~ "A unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bugreport."
#~ msgstr ""
#~ "A frissítés megtervezése közben feloldhatatlan probléma lépett fel.\n"
#~ "\n"
#~ "Jelentse ezt a hibát az \"update-manager\" csomaghoz és vegye be a "
#~ "/var/log/dist-upgrade/ könyvtárban található fájlokat a hibajelentésbe."

#~ msgid "Fetching backport of '%s'"
#~ msgstr "\"%s\" visszaportolt változatának letöltése"

#~ msgid "changes"
#~ msgstr "módosítás"

#~ msgid "Software Channel"
#~ msgstr "Szoftvercsatorna"

#~ msgid "(Source Code)"
#~ msgstr "(Forráskód)"

#~ msgid "<b>Automatic updates</b>"
#~ msgstr "<b>Automatikus frissítések</b>"

#~ msgid "<b>Internet</b>"
#~ msgstr "<b>Internet</b>"

#~ msgid "D_elete downloaded software files:"
#~ msgstr "L_etöltött csomagok törlése:"

#~ msgid "Download from:"
#~ msgstr "Letöltés innen:"

#~ msgid "Import the public key from a trusted software provider"
#~ msgstr "Megbízható szoftverszolgáltató publikus kulcsának importálása"

#~ msgid ""
#~ "Only security updates from the official Ubuntu servers will be installed "
#~ "automatically"
#~ msgstr ""
#~ "Csak az Ubuntu kiszolgálóiról származó biztonsági frissítések kerülnek "
#~ "automatikusan telepítésre."

#~ msgid "Restore _Defaults"
#~ msgstr "_Alapértelmezés visszaállítása"

#~ msgid "Restore the default keys of your distribution"
#~ msgstr "A disztribúcióra jellemző alapértelmezett kulcsok visszaállítása"

#~ msgid "Software Sources"
#~ msgstr "Szoftverforrások"

#~ msgid "_Install security updates without confirmation"
#~ msgstr "_Biztonsági frissítések telepítése megerősítés nélkül"

#~ msgid "<b>Components:</b>"
#~ msgstr "<b>Összetevők:</b>"

#~ msgid "<b>Distribution:</b>"
#~ msgstr "<b>Disztribúció:</b>"

#~ msgid "<b>Type:</b>"
#~ msgstr "<b>Típus:</b>"

#~ msgid "APT line:"
#~ msgstr "APT sor:"

#~ msgid "Edit Source"
#~ msgstr "Forrás szerkesztése"

#~ msgid "_Add Source"
#~ msgstr "_Forrás hozzáadása"

#~ msgid "Configure the sources for installable software and updates"
#~ msgstr "Telepíthető szoftverek és frissítések forrásának beállítása"

#~ msgid "http://changelogs.ubuntu.com/changelogs/pool/%s/%s/%s/%s_%s/changelog"
#~ msgstr ""
#~ "http://changelogs.ubuntu.com/changelogs/pool/%s/%s/%s/%s_%s/changelog"

#~ msgid "Ubuntu 6.10 'Edgy Eft'"
#~ msgstr "Ubuntu 6.10 \"Edgy Eft\""

#~ msgid "Community maintained"
#~ msgstr "Közösségi karbantartású"

#~ msgid "Restricted software"
#~ msgstr "Nem-szabad"

#~ msgid "Cdrom with Ubuntu 6.10 'Edgy Eft'"
#~ msgstr "Az Ubuntu 6.10 \"Edgy Eft\" CD-ROM"

#~ msgid "Ubuntu 6.06 LTS 'Dapper Drake'"
#~ msgstr "Ubuntu 6.06 LTS \"Dapper Drake\""

#~ msgid "Canonical supported Open Source software"
#~ msgstr "A Canonical által támogatott nyílt forrású szoftverek"

#~ msgid "Community maintained (universe)"
#~ msgstr "Közösségi karbantartású (universe)"

#~ msgid "Community maintained Open Source software"
#~ msgstr "Közösségi karbantartású nyílt forrású szoftverek"

#~ msgid "Non-free drivers"
#~ msgstr "Nem-szabad meghajtók"

#~ msgid "Restricted software (Multiverse)"
#~ msgstr "Nem-szabad szoftverek (Multiverse)"

#~ msgid "Cdrom with Ubuntu 6.06 LTS 'Dapper Drake'"
#~ msgstr "Az Ubuntu 6.06 LTS \"Dapper Drake\"-et tartalmazó CD-ROM"

#~ msgid "Ubuntu 5.10 'Breezy Badger'"
#~ msgstr "Ubuntu 5.10 'Breezy Badger'"

#~ msgid "Cdrom with Ubuntu 5.10 'Breezy Badger'"
#~ msgstr "Az Ubuntu 5.10 \"Breezy Badger\"-t tartalmazó CD-ROM"

#~ msgid "Ubuntu 5.10 Security Updates"
#~ msgstr "Ubuntu 5.10 biztonsági frissítések"

#~ msgid "Ubuntu 5.10 Updates"
#~ msgstr "Ubuntu 5.10 frissítések"

#~ msgid "Ubuntu 5.10 Backports"
#~ msgstr "Ubuntu 5.10 visszaportolt csomagok"

#~ msgid "Ubuntu 5.04 'Hoary Hedgehog'"
#~ msgstr "Ubuntu 5.04 \"Hoary Hedgehog\""

#~ msgid "Cdrom with Ubuntu 5.04 'Hoary Hedgehog'"
#~ msgstr "Az Ubuntu 5.04 \"Hoary Hedgehog\"-ot tartalmazó CD-ROM"

#~ msgid "Ubuntu 5.04 Security Updates"
#~ msgstr "Ubuntu 5.04 biztonsági frissítések"

#~ msgid "Ubuntu 5.04 Updates"
#~ msgstr "Ubuntu 5.04 frissítések"

#~ msgid "Ubuntu 5.04 Backports"
#~ msgstr "Ubuntu 5.04 visszaportolt csomagok"

#~ msgid "Ubuntu 4.10 'Warty Warthog'"
#~ msgstr "Ubuntu 4.10 \"Warty Warthog\""

#~ msgid "Community maintained (Universe)"
#~ msgstr "Közösségi karbantartású (Universe)"

#~ msgid "Non-free (Multiverse)"
#~ msgstr "Nem-szabad (Multiverse)"

#~ msgid "Cdrom with Ubuntu 4.10 'Warty Warthog'"
#~ msgstr "Az Ubuntu 4.10 \"Warty Warthog\"-ot tartalmazó CD-ROM"

#~ msgid "No longer officially supported"
#~ msgstr "Hivatalosan már nem támogatott"

#~ msgid "Ubuntu 4.10 Security Updates"
#~ msgstr "Ubuntu 4.10 biztonsági frissítések"

#~ msgid "Ubuntu 4.10 Updates"
#~ msgstr "Ubuntu 4.10 frissítések"

#~ msgid "Ubuntu 4.10 Backports"
#~ msgstr "Ubuntu 4.10 visszaportolt csomagok"

#~ msgid "http://packages.debian.org/changelogs/pool/%s/%s/%s/%s_%s/changelog"
#~ msgstr "http://packages.debian.org/changelogs/pool/%s/%s/%s/%s_%s/changelog"

#~ msgid "Debian 3.1 \"Sarge\""
#~ msgstr "Debian 3.1 \"Sarge\""

#~ msgid "http://security.debian.org/"
#~ msgstr "http://security.debian.org/"

#~ msgid "Debian 3.1 \"Sarge\" Security Updates"
#~ msgstr "Debian 3.1 \"Sarge\" biztonsági frissítések"

#~ msgid "Debian \"Etch\" (testing)"
#~ msgstr "Debian \"Etch\" (tesztelés alatt)"

#~ msgid "http://http.us.debian.org/debian/"
#~ msgstr "http://http.us.debian.org/debian/"

#~ msgid "Debian \"Sid\" (unstable)"
#~ msgstr "Debian \"Sid\" (instabil)"

#~ msgid "DFSG-compatible Software with Non-Free Dependencies"
#~ msgstr "DFSG-kompatibilis szoftver nem-szabad függőségekkel"

#~ msgid "Non-DFSG-compatible Software"
#~ msgstr "Nem DFSG-kompatibilis szoftver"

#~ msgid "Downloading file %li of %li with unknown speed"
#~ msgstr "%li. fájl letöltése, összesen: %li, ismeretlen sebességgel"

#~ msgid "Cancel _Download"
#~ msgstr "Letöltés _megszakítása"

#~ msgid "Could not find any upgrades"
#~ msgstr "Nincs elérhető frissítés"

#~ msgid "Your system has already been upgraded."
#~ msgstr "Az Ön rendszere már frissítve lett."

#~ msgid "Cannot install all available updates"
#~ msgstr "Nem telepíthető minden rendelkezésre álló frissítés"

#~ msgid ""
#~ "<big><b>Examining your system</b></big>\n"
#~ "\n"
#~ "Software updates correct errors, eliminate security vulnerabilities and "
#~ "provide new features."
#~ msgstr ""
#~ "<big><b>A rendszer elemzése</b></big>\n"
#~ "\n"
#~ "A szoftverfrissítések programhibákat javítanak, megszüntetik a biztonsági "
#~ "sebezhetőségeket és új szolgáltatásokat biztosítanak."

#~ msgid ""
#~ "Some updates require the removal of further software. Use the function "
#~ "\"Mark All Upgrades\" of the package manager \"Synaptic\" or run \"sudo apt-"
#~ "get dist-upgrade\" in a terminal to update your system completely."
#~ msgstr ""
#~ "Néhány frissítés további szoftverek eltávolítását igényli. Használja a "
#~ "Synaptic csomagkezelő \"Minden frissítés kijelölése\" funkcióját, vagy "
#~ "futtassa terminálból a \"sudo apt-get dist-upgrade\" parancsot a rendszer "
#~ "teljes frissítéshez."

#~ msgid "The following updates will be skipped:"
#~ msgstr "A következő frissítések ki lesznek hagyva:"

#~ msgid "Download is complete"
#~ msgstr "A letöltés befejeződött"

#~ msgid "The upgrade aborts now. Please report this bug."
#~ msgstr "A frissítés félbeszakadt. Kérem, jelentse ezt hibaként."

#~ msgid "Upgrading Ubuntu"
#~ msgstr "Az Ubuntu frissítése"

#~ msgid "Hide details"
#~ msgstr "Részletek elrejtése"

#~ msgid "Show details"
#~ msgstr "Részletek megjelenítése"

#~ msgid "Only one software management tool is allowed to run at the same time"
#~ msgstr "Csak egy szoftverkezelő eszköz futhat egyidejűleg"

#~ msgid ""
#~ "Please close the other application e.g. 'aptitude' or 'Synaptic' first."
#~ msgstr ""
#~ "Zárja be a másik alkalmazást, például az aptitude vagy Synaptic programokat."

#~ msgid "<b>Channels</b>"
#~ msgstr "<b>Csatornák</b>"

#~ msgid "<b>Keys</b>"
#~ msgstr "<b>Kulcsok</b>"

#~ msgid "Installation Media"
#~ msgstr "Telepítő média"

#~ msgid "Software Preferences"
#~ msgstr "Szoftver beállításai"

#~ msgid "<b>Channel</b>"
#~ msgstr "<b>Csatorna</b>"

#~ msgid "<b>Components</b>"
#~ msgstr "<b>Összetevők</b>"

#~ msgid "Add Channel"
#~ msgstr "Csatorna hozzáadása"

#~ msgid "_Add Channel"
#~ msgid_plural "_Add Channels"
#~ msgstr[0] "Csatorna hozzá_adása"

#~ msgid "_Custom"
#~ msgstr "_Egyéni"

#~ msgid "Ubuntu 6.06 LTS Security Updates"
#~ msgstr "Ubuntu 6.06 LTS biztonsági frissítések"

#~ msgid "Ubuntu 6.06 LTS Updates"
#~ msgstr "Ubuntu 6.06 LTS frissítések"

#~ msgid "Ubuntu 6.06 LTS Backports"
#~ msgstr "Ubuntu 6.06 LTS visszaportolt csomagok"

#~ msgid "Ubuntu 6.06 LTS"
#~ msgstr "Ubuntu 6.06 LTS"

#~ msgid "Some software no longer officially supported"
#~ msgstr "Néhány szoftver már nincs hivatalosan támogatva"

#~ msgid "About %li seconds remaining"
#~ msgstr "Kb. %li másodperc van hátra"

#~ msgid "Source Code"
#~ msgstr "Forráskód"

#~ msgid "Add Cdrom"
#~ msgstr "_CD-ROM hozzáadása"

#~ msgid "Source code"
#~ msgstr "Forráskód"

#~ msgid "_Download updates automatically, but do not install them"
#~ msgstr "Frissítések automatikus _letöltése, a telepítésük nélkül"

#~ msgid ""
#~ "<b><big>The information about available software is out-of-date</big></b>\n"
#~ "\n"
#~ "To install software and updates from newly added or changed sources, you "
#~ "have to reload the information about available software.\n"
#~ "\n"
#~ "You need a working internet connection to continue."
#~ msgstr ""
#~ "<b><big>Az elérhető szoftverekkel kapcsolatos információk "
#~ "elévültek</big></b>\n"
#~ "\n"
#~ "Szoftverek és frissítések telepítéséhez újonnan felvett vagy módosított "
#~ "forrásokból, újra le kell töltenie az elérhető szoftverekkel kapcsolatos "
#~ "információkat.\n"
#~ "\n"
#~ "A folytatáshoz működő hálózati kapcsolatra van szükség."

#~ msgid ""
#~ "<big><b>Enter the complete APT line of the repository that you want to add "
#~ "as source</b></big>\n"
#~ "\n"
#~ "The APT line includes the type, location and components of a repository, for "
#~ "example <i>\"deb http://ftp.debian.org sarge main\"</i>."
#~ msgstr ""
#~ "<big><b>Adja meg a forrásként felvenni kívánt tároló teljes APT "
#~ "sorát</b></big>\n"
#~ "\n"
#~ "Az APT sor tartalmazza a tároló típusát, helyét és összetevőit, például "
#~ "<i>\"deb http://ftp.debian.org sarge main\"</i>."

#~ msgid "Run in a special upgrade mode"
#~ msgstr "Futtatás speciális frissítő módban"

#~ msgid "None"
#~ msgstr "Nincs"

#~ msgid "%s updates"
#~ msgstr "%s frissítések"

#~ msgid "%s (%s)"
#~ msgstr "%s (%s)"

#~ msgid "Nearest server"
#~ msgstr "Legközelebbi kiszolgáló"

#~ msgid "Active"
#~ msgstr "Aktív"

#~ msgid ""
#~ "The upgrade can use the network to check the latest updates and to fetch "
#~ "packages that are not on the current CD.\n"
#~ "If you have fast or inexpensive network access you should answer 'Yes' here. "
#~ "If networking is expensive for you choose 'No'."
#~ msgstr ""
#~ "A frissítés használhatja a hálózatot a legújabb frissítések ellenőrzésére és "
#~ "letöltheti a jelenlegi CD-n el nem érhető csomagokat.\n"
#~ "Ha gyors vagy olcsó hálózati kapcsolattal rendelkezik, válaszoljon igennel. "
#~ "Ha a hálózat elérése drásga, válaszoljon nemmel."

#~ msgid "Changes and description of the update"
#~ msgstr "A frissítés módosításai és leírása"

#~ msgid "<b>CDROM/DVD</b>"
#~ msgstr "<b>CD-ROM/DVD</b>"

#~ msgid ""
#~ "<i>To improve the user experience of Ubuntu please take part in the "
#~ "popularity contest. If you do so the list of installed software and how "
#~ "often it was used will be collected and sent anonymously to the Ubuntu "
#~ "project  on a weekly basis.\n"
#~ "\n"
#~ "The results are used to improve the support for popular applications and to "
#~ "rank applications in the search results.</i>"
#~ msgstr ""
#~ "<i>Az Ubuntu által biztosított felhasználói élmény javítása érdekében kérjük "
#~ "vegyen részt a népszerűségi versenyben. Ha így dönt, akkor a telepített "
#~ "szoftverek listája és azok használatának gyakorisága hetente összegyűjtésre "
#~ "és névtelenül elküldésre kerül az Ubuntu projektnek.\n"
#~ "\n"
#~ "Az eredmények a népszerű alkalmazások támogatásának javításához és a "
#~ "keresési eredmények rangsorolásához kerülnek felhasználásra.</i>"

#~ msgid "Statistics"
#~ msgstr "Statisztika"

#~ msgid "Submit statistical information"
#~ msgstr "Statisztikai információk beküldése"

#~ msgid "Third Party"
#~ msgstr "Harmadik fél"

#~ msgid "Proprietary drivers for devices"
#~ msgstr "Szabadalmazott eszközmeghajtók"

#~ msgid "Proprietary drivers for devices "
#~ msgstr "Szabadalmazott eszközmeghajtók "

#~ msgid "Software restricted by copyright or legal issues"
#~ msgstr "Szerzői vagy egyéb jogi problémák miatt korlátozott szoftver"

#~ msgid "Backported updates"
#~ msgstr "Visszaportolt frissítések"

#~ msgid ""
#~ "<span weight=\"bold\" size=\"x-large\">Upgrading to Ubuntu 6.10</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"x-large\">Frissítés az Ubuntu 6.10 "
#~ "változatra</span>"

#~ msgid "Important security updates of Ubuntu"
#~ msgstr "Az Ubuntu fontos biztonsági frissítései"

#~ msgid "Updates of Ubuntu"
#~ msgstr "Az Ubuntu frissítései"

#~ msgid "Normal updates"
#~ msgstr "Normális frissítések"
